markdown/org/docs/patterns/hortensia/fabric/en.md

```diff
@@ -1,5 +1,16 @@
-<Fixme>
 
-Complete documentation for Hortensia
+### Main Fabric
 
-</Fixme>
+The key thing across all is non-stretch, durable and strong enough to hold items.
+
+Your main fabric choice ultimately comes down to your personal style and what you wish to use your handbag for. Due to Hortensia having a size option it can also be enlarged and used for things such as a gym bag which does not need to be as fancy as an upmarket handbag. So you may just simply want to go with **Cottons**, **Linens** or have a little fun with **Printed Cottons**. If you want something more fancy you can try going for **Brocades** and other luxe fabrics. If you want something that will be very durable you can try **Upholstery fabrics**. You can also try **Leather** but this will be harder to sew especially if it is thick so only use it if you know what you are doing. Remember if you wish to use it all the time, practically over design as the goal should be to make it last rather than make it fancy. If you do not care about the look at all then this may be an oppurtunity to use up fabric from your stash that you otherwise would not such as that ugly fabric you can never find a use for.
+
+<Note>
+
+If you find that the fabric you want to use is a bit too thin or flimsy you can interface the pieces to give then strength.
+
+</Note>
+
+### Lining Fabric
+
+On the flipside the lining does not need to be as strong and simply needs to cover the raw seams and protect the outer fabric. If you want something fancy you can go for **Silks** and **Satins** but **Cotton** linings will do just as well. Bare in mind on what you intend to store in the bag. If you intend to carry a lot you should go for something less-prone to tearing, if you are only using it for knick-kancks and small personal items you can get away with fancier fabrics such as silks. The lining is also an oppurtunity to use up any fabrics in your stash that you don't like or don't have a use for as it will be hidden in the final product and only you will know/see.
```

Comment:
```suggestion
On the flipside the lining does not need to be as strong and simply needs to cover the raw seams and protect the outer fabric. If you want something fancy you can go for **Silks** and **Satins** but **Cotton** linings will do just as well. Bear in mind what you intend to store in the bag. If you intend to carry a lot you should go for something less prone to tearing, if you are only using it for knick-knacks and small personal items you can get away with fancier fabrics such as silks. The lining is also an opportunity to use up any fabrics in your stash that you don't like or don't have a use for as it will be hidden in the final product and only you will know/see.
```